tests/integration/docker/info.bats

```diff
@@ -31,6 +31,8 @@ setup() {
 @test "Container info" {
 	container=$(random_name)
 	$DOCKER_EXE run -itd --name $container busybox
-	$DOCKER_EXE info| grep "Containers: 1"
+	run $DOCKER_EXE inspect --type container --format '{{ .State.Status }}' $container
+	[ "$status" -eq 0 ]
+	[ "$output" = "running" ]
```

Comment:
Nice idea, but this test is supposed to be testing `docker info` ;-)

Maybe add back in the call to `docker info` and keep the extra tests so we have some degree of assurance that the container state we're checking in `info` relates to the one we created?